docs/Doxyfile

```diff
@@ -189,7 +189,7 @@ ENUM_VALUES_PER_LINE   = 1
 TREEVIEW_WIDTH         = 250
 EXT_LINKS_IN_WINDOW    = NO
 FORMULA_FONTSIZE       = 10
-USE_MATHJAX            = NO
```

Comment:
If someone wants to add a new block that does not already have a CircuiTikZ schematic, do we document somewhere how they can generate one?

Reply:
I'll add something to the developer guide

Reply:
@menon-karthik plz have a look!

Reply:
Thanks!